packages/abi-coder/src/utils/constants.ts

```diff
@@ -33,7 +33,7 @@ export const ENCODING_V1 = '1';
  */
 export const WORD_SIZE = 8;
 export const BYTES_32 = 32;
-export const UTXO_ID_LEN = BYTES_32 + 1;
+export const UTXO_ID_LEN = BYTES_32 + 2;
```

Comment:
https://github.com/FuelLabs/fuel-vm/pull/698/files

Previous to VM 0.48, UTXOs IDs had 33 bytes:
- The first 32 bytes is the Transaction ID that created the UTXO
- The following 1 byte (u8) is the output index where the `OutputCoin` that originated this UTXO was placed within the `outputs` of the Transaction.

On VM 0.48 the output index changed from u8 to u16, so now UTXO IDs have 34 bytes instead of 33.